packages/concerto-cto/lib/parser.pegjs

```diff
@@ -1565,6 +1619,8 @@ SourceElement
   / EnumDeclaration
   / ConceptDeclaration
   / ScalarDeclaration
-
+  / MapDeclaration
+  / MapPropertyKeyDeclaration
```

Comment:
I think that this means that I could declare an orphaned Key or Value without an enclosing MapDeclaration?

Reply:
You are correct